tests/specs/task/filter/deno_workspace_order.out

```diff
@@ -1,4 +1,4 @@
-Task dev echo '@foo/baz'
-@foo/baz
 Task dev echo '@foo/bar'
 @foo/bar
+Task dev echo '@foo/baz'
```

Comment:
Note to self: I need to take a look at these tomorrow. I think the order of tasks is wrong here but maybe I made it too confusing with the fixtures.

Reply:
They didn't seem to depend on each other, and I believe were printed in reversed order

Reply:
The tasks itself yeah, but the packages do. When package A depends on B, the matching tasks from B must be executed first.